### ISSUE TITLE
Accessibility: The keyboard focus comes to container element of button groups and are no-op when activated by pressing enter or spacebar.  

### ISSUE THREAD

**Issue by ericmaino** [microsoft/BotFramework-WebChat#3086]
![image](https://user-images.githubusercontent.com/1005609/78628634-8f479400-7849-11ea-9fec-754c3dbe4d18.png)

**Expected**
The keyboard focus should come to actionable elements which can be invoked by pressing enter/spacebar. There should be no tab stop on non-actionable elements. They should be removed from tab order.

**Actual**
The keyboard focus is present on container element which contains a single choice button or group of multiple choice buttons. 

**WCAG Violation**
Focus Order 2.4.3
https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html

Keyboard: 2.1.1
https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html

**Comment by corinagum** [comment]
These are accessibility issues of v3 of Web Chat, which has been deprecated and we no longer support unless there are security risks. For accessible Web Chat, please [migrate to v4](https://github.com/Microsoft/BotFramework-WebChat#migrating-from-web-chat-v3-to-v4).

**Comment by compulim** [comment]
This is a bug on Adaptive Cards we talked recently.

The AC card (gray bubble in your screenshot) can be set to be clickable. I mean, the whole card can be set to clickable.

If the card is clickable, it should be focusable.

Unfortunately, there is a bug in AC that even the card is not clickable, it is continued to be focusable. That is the term "rogue tabindex" we used in our offline chat.

Thus, you are seeing the focus didn't jump into the button but the AC card itself. It is because they added a `tabindex="0"` to the card even it is not clickable.
